### DISCUSSION TITLE
Google Photos API Outstanding Bugs

### DISCUSSION THREAD

**Discussion by gilesknap** [gilesknap/gphotos-sync#347] [Announcements]
## Welcome!
  Please share your experiences with the Google Photos API bugs that have gone unresolved for a few years.

My list of the bugs affecting this project are here #119

To vote on a poll see #359

I would like to use this forum for:

- ideas to work around these issues
- ideas for getting Google to look at fixing them
- any other useful thoughts

**Comment by pataquets** [comment]
I [added an "Outstanding issues" section](https://en.wikipedia.org/w/index.php?title=Google_Photos&oldid=1074880066#Outstanding_issues) on the Google Photos [Wikipedia article](https://en.wikipedia.org/wiki/Google_Photos) some time ago, to at least "make the shame public", but some Google fanboy (as self-declared in its profile) quickly [reverted it](https://en.wikipedia.org/w/index.php?title=Google_Photos&diff=prev&oldid=1074972752) without any discussion/debate (which is against WP spirit).

I still think that making those bugs clearly visible on WP (which lots of people check for comparing services) would have some effect, but I have not pushed it anymore.

**Reply by gilesknap** [reply]
That is a good idea. I'm not familiar with contributing to WP. Is there any way to avoid such reversion with enough contributors on your side?

**Reply by osr00** [reply]
> I [added an "Outstanding issues" section](https://en.wikipedia.org/w/index.php?title=Google_Photos&oldid=1074880066#Outstanding_issues) on the Google Photos [Wikipedia article](https://en.wikipedia.org/wiki/Google_Photos) some time ago, to at least "make the shame public", but some Google fanboy (as self-declared in its profile) quickly [reverted it](https://en.wikipedia.org/w/index.php?title=Google_Photos&diff=prev&oldid=1074972752) without any discussion/debate (which is against WP spirit).
> 
> I still think that making those bugs clearly visible on WP (which lots of people check for comparing services) would have some effect, but I have not pushed it anymore.

The comment from the reverter is really annoying. State another source. Your change is really clear, and objective. There is no doubt that this issues really exists! I reverted the revert in WP. Currently it should be online again. We will see what happens.

**Reply by pataquets** [reply]
Great to hear that, @osr00 and welcome to the Wikipedia!
[This page](https://en.wikipedia.org/wiki/Help:Introduction_to_policies_and_guidelines/1) is a good starting poiont for contributing to WP. From there, there are some general guidelines and also strict policies about what's considered true (assume good faith, cite your sources, etc.) Ideally, conflicts should be carried over the article's own discussion page to [reach consensus](https://en.wikipedia.org/wiki/Wikipedia:Consensus). If everything else fails, there is also a specific help page about [edit wars](https://en.wikipedia.org/wiki/Wikipedia:Edit_warring), but that should be considered a last resort measure.
@gilesknap The edit wars article and related ones explain it. I guess that people watching the Google Groups threads would agree with the edition, too. :smile:
